### PR TITLE
Update add_controller how-to for JIMM v2

### PR DESCRIPTION
**Pull request by kian99**
Update the add_controller how-to for JIMM v2.

CSS-3368

### CODE REVIEW

File: how-to/add_controller.rst [hunk 1]
```diff
@@ -69,9 +62,9 @@ Deploy controller
 
 2. Now we are ready to bootstrap a controller. Please note the constraints here are the ones used for production JAAS services and should be suitable for most loads. If it is anticipated that the JAAS system will have a different model profile then we encourage you to determine the appropriate constraints for your system: 
 
-    ``juju bootstrap --no-gui --bootstrap-constraints="root-disk=50G cores=8 mem=8G" --config identity-url=$CANDID --config allow-model-access=true --config public-dns-address=$DNS:443 $CLOUD/$REGION $NAME``
+    ``juju bootstrap --bootstrap-constraints="root-disk=50G cores=8 mem=8G" --config login-token-refresh-url=https://$JIMM/.well-known/jwks.json --config allow-model-access=true --config public-dns-address=$DNS:443 $CLOUD/$REGION $NAME``
 
-3. Next the controller should be put into HA mode: 
+3. Next the controller should be put into HA mode (optional): 
 
     ``juju enable-ha``
```
Comment (ale8k) [inline how-to/add_controller.rst:69]: Does this rst stuff support markdown like things? I.e., > to put notes?

Comment (kian99) [reply how-to/add_controller.rst:69]: Yeah, there's a primer here https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-primer

File: how-to/add_controller_no_dns.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    +----------------------+----------------------+
+    | Cloud Region         | $REGION              |
+    +----------------------+----------------------+
+    | Candid URL           | $CANDID              |
+    +----------------------+----------------------+
+    | JIMM URL             | $JIMM                |
+    +----------------------+----------------------+
+
+
+2. Now we are ready to bootstrap a controller. Please note the constraints here are the ones used for production JAAS services and should be suitable for most loads. If it is anticipated that the JAAS system will have a different model profile then we encourage you to determine the appropriate constraints for your system:
```
Comment (ale8k) [inline how-to/add_controller_no_dns.rst:51]: You say model constraints and then say model profile, shouldn't this say "will have a different set of model constraints"?

**Review by ale8k — APPROVED** [review]

File: how-to/add_controller.rst (hunk 1 above)
Comment (ale8k) [reply how-to/add_controller.rst:69]: can't see the > equivalent but if we got it, let's use it for notes

File: how-to/add_controller_no_dns.rst (hunk 2 above)
Comment (kian99) [reply how-to/add_controller_no_dns.rst:51]: Reused the language from the add_controller.rst and I think the meaning is clear.

**Pull request merged**